keyboards/specskeys/info.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This file needs to be filled out, or it will not work correctly in the Configurator.

Reply:
I do not seem to manage to find an appropriate howto on this topic.

> qmk_firmware [dev_branch] » qmk c2json -kb specskeys -km default keyboards/specskeys/keymaps/default/keymap.c -o ~/Documents/info.json

does not deliver usable results. 

> https://qmk.fm/converter/

does not make it better.

What do I have to do in order to create one info.json ?